src/platform/silabs/ConnectivityManagerImpl_WIFI.cpp

```diff
@@ -220,6 +220,28 @@ void ConnectivityManagerImpl::_OnWiFiStationProvisionChange()
     DeviceLayer::SystemLayer().ScheduleWork(DriveStationState, NULL);
 }
 
+#if CHIP_DEVICE_CONFIG_ENABLE_SED
+CHIP_ERROR ConnectivityManagerImpl::_GetSEDIntervalsConfig(ConnectivityManager::SEDIntervalsConfig & SEDIntervalsConfig)
+{
+    // For now Wi-Fi uses DTIM power save mode
+    SEDIntervalsConfig.ActiveIntervalMS = chip::System::Clock::Milliseconds32(50);
+    SEDIntervalsConfig.IdleIntervalMS   = chip::System::Clock::Milliseconds32(300);
+    return CHIP_NO_ERROR;
+}
+
+CHIP_ERROR ConnectivityManagerImpl::_SetSEDIntervalsConfig(const ConnectivityManager::SEDIntervalsConfig & intervalsConfig)
+{
+    // not required
+    return CHIP_NO_ERROR;
+}
+
+CHIP_ERROR ConnectivityManagerImpl::_RequestSEDActiveMode(bool onOff, bool delayIdle)
+{
+    // not required
+    return CHIP_NO_ERROR;
```

Comment:
idem

Reply:
no need of setting active mode in WiFi case, we receive packets on the DTIM interval. The device receives packets when the command is executed

Reply:
see above for thread.

Reply:
Resolving based on discussion